src/components/assets/Assets.vue

```diff
@@ -95,6 +94,22 @@ export default defineComponent({
 
     const isShibuya = computed(() => currentNetworkIdx.value === endpointKey.SHIBUYA);
 
+    // v2
+    const xcmAssets = computed(() => store.getters['assets/getAllAssets']);
```

Comment:
This value returns an empty array in H160 mode.
![image](https://user-images.githubusercontent.com/92044428/178938057-37a8c0d3-2c4c-461e-8378-ae2fd11bb009.png)

Reply:
https://github.com/AstarNetwork/astar-apps/blob/90f9100828691f8130db50ee5f181fabf431be0c/src/components/assets/Assets.vue#L105

Could you share with me how to reproduce an infinite loop? It seems like the issue has been resolved after removing this if statement.

<img width="1274" alt="image" src="https://user-images.githubusercontent.com/92044428/178940229-d6b80760-ce2a-4e2e-9cd9-bd9e16a8b422.png">

Reply:
@bobo-k2 
Please take a look at [this](https://github.com/AstarNetwork/astar-apps/pull/464) PR.

Reply:
> https://github.com/AstarNetwork/astar-apps/blob/90f9100828691f8130db50ee5f181fabf431be0c/src/components/assets/Assets.vue#L105
> 
> Could you share with me how to reproduce an infinite loop? It seems like the issue has been resolved after removing this if statement.
> 

Just remove if statement and switch to Metamask

Reply:
> @bobo-k2 It works as expected on my end after merging [this](https://github.com/AstarNetwork/astar-apps/pull/464) PR. Please let me know if you encounter the same issue again.

Thanks @impelcrypto works great